wd.plugin.zsh

```diff
@@ -7,4 +7,9 @@
 #
 # @github.com/mfaerevaag/wd
 
-eval "wd() { source '${0:A:h}/wd.sh' }"
+source ${0:A:h}/wd.sh
+
+# Register the function as a Zsh widget
+zle -N wd_browse
+# Bind the widget to a key combination
+bindkey '^G' wd_browse
```

Comment:
They can put `bindkey '^G' wd_browse` in their ~/.zshrc

Reply:
Ah okay, so it wouldn't be shipped to those users by default?

Reply:
Yes it would, do you want me to delete it and put it in the readme instead?
Its currently directly in the plugin file

Reply:
Yeah I think this would be less controversial. I just don't want to mess up a big community of users by adding a keybind that could conceivably be bound to something else

Registering the zsh widget is a good idea because then it's just one line in someone's `.zshrc`